helpers/cli.py

```diff
@@ -44,14 +44,15 @@ def colored_print(cls, message, color=NO_COLOR):
 
     @classmethod
     def colorize(cls, message, color=NO_COLOR):
-        return '{}{}{}'.format(color, message, cls.NO_COLOR)
+        return f'{color}{message}{cls.NO_COLOR}'
```

Comment:
If I run:
```bash
python2.7 run.py
```
I get this:
```
Traceback (most recent call last):
  File "run.py", line 6, in <module>
    from helpers.cli import CLI
  File "/tmp/kobo-install/helpers/cli.py", line 47
    return f'{color}{message}{cls.NO_COLOR}'
                                           ^
SyntaxError: invalid syntax
```
Looks like we need to keep `CLI` friendly with <3.5 to at least display the warning:
```
josh@work:/tmp/kobo-install$ git checkout origin/master -- helpers/cli.py
josh@work:/tmp/kobo-install$ python2.7 run.py
╔════════════════════════════════════════════════════════════════════╗
║                                                                    ║
║ Your Python version has reached the end of its life.Please upgrade ║
║ it as it is not maintained anymore.                                ║
║                                                                    ║
╚════════════════════════════════════════════════════════════════════╝
```